src/components/reportingCards/ReportingCard.js

```diff
@@ -145,7 +145,7 @@ function GrowerReportingCard(props) {
     setOpen(true);
   }
 
-  log.warn('icon', icon);
+  // log.warn('icon', icon);
```

Comment:
What do you think about removing it completely, rather than commenting it out? I would prefer to have it removed so we don't have useless comments in the code.

Reply:
Sure thing. I didn't know if we would need it for future tests or something; that's why I didn't delete it, but yeah makes sense to remove it altogether.